### ISSUE TITLE
drop down accuracy during training

### ISSUE THREAD

**Issue by azizyemen1** [bubbliiiing/arcface-pytorch#33]
i have one problem during traing CASIA-WebFace ,the accuracy is droping dwon i used same model parameters

**Comment by bubbliiiing** [comment]
can u show me the accuracy line？

**Comment by azizyemen1** [comment]
> CASIA-WebFace
![ACC](https://github.com/bubbliiiing/arcface-pytorch/assets/45854677/8ddad4b8-8184-4012-9dc4-7df7f9a65379
I reduced the lr then the accuracy  became 0 for all epochs
![LR](https://github.com/bubbliiiing/arcface-pytorch/assets/45854677/da723fa6-3f67-4738-80ef-8ec25ad178f6)

**Comment by bubbliiiing** [comment]
not this. accuracy of lfw is more important

**Comment by juren1** [comment]
> not this. accuracy of lfw is more important不是这个。LFW的准确性更为重要

为什么我在训练的时候，训练集验证集正确率为0.5左右，而lfw可以达到0.99，这数据正常吗？还是说训练集验证集正确率数据要经过什么转换才可以真实的数据

**Comment by juren1** [comment]
> i have one problem during traing CASIA-WebFace ,the accuracy is droping dwon i used same model parameters我在训练CASIA-WebFace期间有一个问题，使用相同的模型参数时，准确性下降

大佬，你这个训练的时候遇到我这个问题了吗？有什么建议吗？希望不吝赐教，谢谢大佬 @azizyemen1 @bubbliiiing
